.travis.yml

```diff
@@ -6,19 +6,13 @@ rust:
 
 os:
   - linux
-  - osx
 
 env:
   global:
     - RUST_BACKTRACE=1
   matrix:
     - FEATURES="unstable"
-    - FEATURES="unstable force-inprocess"
-
-matrix:
-  include:
-    - os: linux
-      env: FEATURES="unstable memfd"
+    - FEATURES="unstable memfd"
```

Comment:
It seems like this part was temporarily edited, according to the history (2f2b1586bd22ef0713ab695748d7ea9ef4966586 and 3facdd3920aeda7d5823b8bb1a928401a2eb98be) . I think I need to revert these commits.

Reply:
Good catch.